src/main/kotlin/com/openlattice/data/storage/PostgresEntityDataQueryService.kt

```diff
@@ -128,6 +128,10 @@ class PostgresEntityDataQueryService(
         }
     }
 
+    /**
+     * Returns a map of requested entity key ids to their data, where their data is represented as a map of property
```

Comment:
Comment should reflect that it's an Iterable<Pair> and not a map anymore :)

Reply:
Right, thanks!